R/snapshot.R

```diff
@@ -91,6 +91,9 @@ the$auto_snapshot_hash <- TRUE
 #'   to the currently active package repositories, as retrieved by
 #'   `getOption("repos")`.
 #'
+#' @param dev Boolean; should development dependencies be captured
```

Comment:
I think it also applies to the type all" Regardless, I'd suggest pointing people to the `?dependencies` for more details.

Reply:
I added the link to the `dependencies()` help into the **See Also** section for more details about handling R package dependencies. Feel free to modify if you think this should be more prominent in `shapshot()`.